### ISSUE TITLE
[Suggestion] A little code change

### ISSUE THREAD

**Issue by PencilMario** [my-otter-self/MAS_noises#1]
*Note: I am using a machine translation so it may be difficult to understand some parts, my apologies X(*

I was going to launch a pull request directly, but the repository is so big that I haven't clone it until now :(

So I'll just talk here about how you can optimize the size 

1. Audio files
When I started translating today, I actually saw that the length of a single file was actually up to an hour! Such a long soundtrack is clearly a waste of space. Just set it to loop when you play it, and it's actually hard to hear that it's looping every 6 minutes.
MAS has a built-in weather sound effect of less than 5M, but again, you can't hear the loop

2. about the play_song() function
I see that you use this function to play noise, but it is designed for BGM and actually forces the player's custom music to stop when it is used, which I don't think is very good at least.
MAS has created a new audio channel to play weather sounds like rain (line 357 of script-ch30.rpy)
So we could have a custom function to play the noise

```renpy
def noice_play(song, fadein=1.2, loop=True, fadeout=1.2, if_changed=False):
    if song is None:
        renpy.music.stop(channel="background", fadeout=fadeout)
    else:
        song = # Change to the play path of your file!!!
        renpy.music.play(
            song,
            channel="background",
            loop=loop,
            synchro_start=True,
            fadein=fadein,
            fadeout=fadeout,
            if_changed=if_changed
        )
```
This way it can be looped to save space and not interrupt the player's custom music :)

**Comment by dreamscached** [comment]
Thank you for suggestion, we'll discuss decreasing file sizes with @my-otter-self, as for the way to play noises &mdash; yes, we'll change to that approach.

**Comment by dreamscached** [comment]
Implemented in 51319b4
